### PR TITLE
move pkg/git to library-go

### PR DESCRIPTION
**Pull request by bparees**

### CODE REVIEW

File: pkg/git/repository_test.go [hunk 1]
```diff
@@ -0,0 +1,149 @@
+package git
+
+import (
+	"testing"
+	"time"
+)
+
+func TestGetRootDir(t *testing.T) {
```
Comment (bparees) [inline pkg/git/repository_test.go]: @mfojtik @deads2k do we run unit tests against library-go?

Comment (bparees) [reply pkg/git/repository_test.go]: @mfojtik @deads2k bump

Comment (mfojtik) [reply pkg/git/repository_test.go]: Yes we do, in Travis

**Comment by deads2k** [comment]
merging per michal's lgtm

**Pull request merged**